orangecontrib/imageanalytics/image_embedder.py

```diff
@@ -193,11 +196,21 @@ def _send_to_server(self, file_paths, image_processed_callback):
         )
 
     def _load_image_or_none(self, file_path):
-        try:
-            image = open_image(file_path)
-        except IOError:
-            log.warning("Image skipped (invalid file path)", exc_info=True)
-            return None
+        parsed_url = urlparse(file_path)
+        if parsed_url.scheme == 'http' or parsed_url.scheme == 'https':
+            response = requests.get(file_path, stream=True)
```

Comment:
That list you reference is quite extensive. Is there someone in the Orange3 core team who could help @earthling2304 with that task, so it would pass review?